frappe/database/database.py

```diff
@@ -891,21 +889,6 @@ def set_value(
 		:param debug: Print the query in the developer / js console.
 		"""
 
-		if _is_single_doctype := not (dn and dt != dn):
```

Comment:
Deprecated APIs/behavior failing silently seems dangerous. Should this be raised instead? 

----

Frappe: Uses dn == dt to check if API is used for Single DocType 
POV: You have a Note with name Note 💀